File: brad-kerr/create-phone-number.js
//return: the numbers in a phone number format
//paramter: an array of 10 integers
//break the array into three different strings based off the index slice needed
//string concat them with the other symbols to display the phone number

function createPhoneNumber(numbers){
let one = numbers.slice(0,3).join('')
let two = numbers.slice(3,6).join('')
let three = numbers.slice(6,10).join('')

return `(${one}) ${two}-${three}`;
}

File: brad-kerr/create-phone-number.py
# Return: the numbers in a phone number format
# Paramter: an array of 10 integers
# Break the array into three different strings based off the index slice needed
# Join them with the other symbols needed to display the phone number correctly

def create_phone_number(n):
    one = ''.join(map(str, n[:3]))
    two = ''.join(map(str, n[3:6]))
    three = ''.join(map(str, n[6:10]))

    return f'({one}) {two}-{three}'
